backend_new/src/dtos/applications/household-member-update.dto.ts

```diff
@@ -15,12 +15,12 @@ export class HouseholdMemberUpdate extends OmitType(HouseholdMember, [
   @Expose()
   @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
   @Type(() => AddressCreate)
-  @ApiProperty({ type: AddressCreate })
+  @ApiPropertyOptional({ type: AddressCreate })
```

Comment:
Curious about the shift here?

Reply:
Good point and the old backend also had it required. I'll remove this and address this in the next PR where the application partner flow is tackled